From f5db37f59f69bccbef9ca6b2a792031bc4d70ee2 Mon Sep 17 00:00:00 2001
From: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>
Date: Fri, 1 Jun 2018 15:15:18 +0530
Subject: [PATCH] saveconfig: fix missing import

$ targetcli /backstores/user:glfs/test saveconfig
NameError: "global name 'errno' is not defined"

Signed-off-by: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>
---
 rtslib/root.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rtslib/root.py b/rtslib/root.py
index 13401e5..7ea06fa 100644
--- a/rtslib/root.py
+++ b/rtslib/root.py
@@ -22,6 +22,7 @@ import os
 import stat
 import json
 import glob
+import errno
 
 from .node import CFSNode
 from .target import Target
-- 
1.8.3.1

